src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
```

Comment:
Ah, okay, makes sense. In that case, it might make sense to make a function called `repositionTooltip` that does the `$timeout` call, as you're trying to make the same reposition call in multiple places, it helps reading. I missed the fact that all the $timeout calls have the same arguments.